test/Carter.Samples.Tests/OpenAPITests.Should_return_Carter_approved_OpenAPI_json.approved.txt

```diff
@@ -229,78 +235,62 @@
   },
   "components": {
     "schemas": {
-      "CastMembers": {
-        "type": "array",
-        "items": {
-          "$ref": "#/components/schemas/CastMember"
-        }
-      },
-      "CastMember": {
-        "type": "object",
-        "properties": {
-          "name": {
-            "type": "string"
-          }
-        },
-        "example": {
-          "name": ""
-        }
-      },
-      "Actors": {
-        "type": "array",
-        "items": {
-          "$ref": "#/components/schemas/Actor"
-        }
-      },
       "Actor": {
-        "required": [
-          "name"
-        ],
         "type": "object",
         "properties": {
           "name": {
-            "minLength": 1,
```

Comment:
We shouldn't lose the validation fields like required and minLength and excluseiveMinimum